Sources/TensorFlow/Layers/Dropout.swift

```diff
@@ -73,3 +73,31 @@ public struct Dropout<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
     }
   }
 }
+
+public struct GaussianDropout<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
+  @noDerivative
+  public let probability: Double
```

Comment:
It doesn't appear this is used currently, unless I'm missing something?

Reply:
To piggy-back on this, the probability should be used and `standardDeviation` should be computed by using this `probability`, see the [Keras implementation](https://github.com/keras-team/keras/blob/master/keras/layers/noise.py#L58),